solr/modules/s3-repository/build.gradle

```diff
@@ -20,7 +20,9 @@ apply plugin: 'java-library'
 description = 'S3 Repository'
 
 dependencies {
-  api project(':solr:core')
+  api(project(':solr:core')) {
+    exclude group: 'org.ow2.asm', module: '*'
```

Comment:
them solr:core shouldn't be publishing that dependency in the first place. At least see if that can be done.